app/repos/exemption.js

```diff
@@ -114,6 +116,22 @@ const handleOrder2023 = (registration, data) => {
   }
 }
 
+/**
+ * @param registration
+ * @param data
+ */
+const setDefaults = (registration, data) => {
+  if (
+    registration.cdo_expiry === null &&
```

Comment:
Are we sure the cdo_expiry will be null as opposed to undefined?

Reply:
did all of these points on an earlier iteration 😆

Reply:
have added checks for undefined and coverage for those branches